Projects/App/Sources/MainNavigationJourney.swift

```diff
@@ -272,7 +273,35 @@ struct MainNavigationJourney: App {
                 }
                 .fullScreenCover(item: $contractsNavigationVm.terminationContract) { contract in
                     let contractConfig: TerminationConfirmConfig = .init(contract: contract)
-                    TerminationViewJourney(configs: [contractConfig])
+                    TerminationFlowNavigation(
+                        configs: [contractConfig],
+                        isFlowPresented: { terminationDismissType in
+                            contractsNavigationVm.terminationContract = nil
+                            switch terminationDismissType {
+                            case .none:
+                                break
+                            case .chat:
+                                contractsNavigationVm.isChatPresented = true
+                            case .openFeedback(let url):
```

Comment:
@sladan-hedvig not really sure how to place this. Also not sure how to make it work for embedded browser. Does not work right now.